libmachine/host.go

```diff
@@ -256,6 +256,15 @@ func (h *Host) Upgrade() error {
 }
 
 func (h *Host) Remove(force bool) error {
+	machineState, err := h.Driver.GetState()
+	if err != nil {
+		return err
+	}
+
+	if machineState != state.Stopped {
```

Comment:
What if the state is `Error`?

Also, would it maybe make sense to skip this check if `force` is set? If I do a `docker rm -f mycontainer`, it kills then removes the container, so it follows that doing a `docker-machine rm -f mymachine` should stop then remove the machine.